components/l10n/browser/locale_helper.cc

```diff
@@ -34,7 +34,7 @@ LocaleHelper* LocaleHelper::GetInstance() {
   return GetInstanceImpl();
 }
 
```

Comment:
This change should not include iOS, thanks

Reply:
@tmancey the original code (current `master`) has both OS_MACOSX and OS_IOS, see
https://github.com/brave/brave-core/blob/master/components/l10n/browser/locale_helper.cc#L37
If I don't have IOS in there, I get an compilation error when building for iOS in Debug about a duplicate symbol `GetInstanceImp` defined in `locale_helper.o` and `locale_helper_ios.o`.

Reply:
Ah yes apologies as was checking in my mobile and missed that. Thanks